### PR TITLE
Judging_Reminder_2017

### PR DESCRIPTION
**Pull request by ultimatelyoctopuses**

### CODE REVIEW

File: mivs/templates/emails/judging_reminder.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+{% if judge.judging_complete %}Our records indicate that you've completed reviews of all of games you've been assigned.  Thanks!  Please let us know if you're available to judge any additional games, as some other judges may find themselves unable to do so.{% else %}Our records indicate that you have completed judging on {{ judge.game_reviews|length }} games, out of the {{ judge.game_reviews|length }} you have been assigned.  Please try to get the remainder evaluated before the deadline, and if you don't think you'll be able to do so, please let us know so that we can assign those games to another judge.{% endif %}
```
Comment (RobRuana) [inline mivs/templates/emails/judging_reminder.txt]: I think the second `{{ judge.game_reviews|length }}` should be `{{ judge.reviews|length }}`. Looks like that was maybe an existing bug, but we should probably fix it while we're making changes here!

Comment (RobRuana) [inline mivs/templates/emails/judging_reminder.txt]: Also "reviews of all ~of~ games you've been assigned" should be "reviews of all **the** games you've been assigned".

**Comment by ultimatelyoctopuses** [comment]
I agree with the changes! Thank you so so so much!

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by RobRuana** [comment]
FYI, this change is now live!

**Pull request merged**